cvat/schema.yml

```diff
@@ -12,6 +12,106 @@ info:
     name: MIT License
     url: https://en.wikipedia.org/wiki/MIT_License
 paths:
+  /api/analytics/reports:
+    get:
+      operationId: analytics_list_reports
+      description: Receive analytics report
+      summary: Method returns analytics report
+      parameters:
+      - in: query
+        name: end_date
+        schema:
+          type: string
+          format: date-time
+        description: Specify the end date for filtering report data.
+      - in: query
+        name: job_id
+        schema:
+          type: integer
+        description: Specify job ID
+      - in: query
+        name: project_id
+        schema:
+          type: integer
+        description: Specify project ID
+      - in: query
+        name: start_date
+        schema:
+          type: string
+          format: date-time
+        description: Specify a start date for filtering report data.
+      - in: query
+        name: task_id
+        schema:
+          type: integer
+        description: Specify task ID
+      tags:
+      - analytics
+      security:
+      - sessionAuth: []
+        csrfAuth: []
+        tokenAuth: []
+      - signatureAuth: []
+      - basicAuth: []
+      responses:
+        '200':
+          description: '...'
+        '404':
+          description: Not found
+    post:
+      operationId: analytics_create_report
+      summary: Creates a analytics report asynchronously and allows to check request
+        status
+      parameters:
+      - in: query
+        name: job_id
+        schema:
+          type: integer
+        description: Specify job ID
+      - in: query
+        name: project_id
+        schema:
+          type: integer
+        description: Specify project ID
```

Comment:
I suggest moving these 2 parameters to the request body. It looks natural to have them there since they describe the request object.